qemu/tests/cfg/numa_opts.cfg

```diff
@@ -1,7 +1,8 @@
 - numa_opts:
   type = numa_opts
-  # paused VMs are enough for our purposes, no need to boot them:
-  paused_after_start_vm = yes
+  Windows:
+      # paused VMs are enough for windows, no need to boot them:
+      paused_after_start_vm = yes
```

Comment:
I'd suggest to boot windows guest as well, but no need to check numa inside guest.

Reply:
Could you explain the reason ?
For windows vm, it is same with before, no need to login vm, just check with monitor.

Reply:
We should test guest can boot up well with the different numa configuration.

Reply:
> We should test guest can boot up well with the different numa configuration.

Ok, I can update it in this patch to cover this test scenario too ~